essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md

```diff
@@ -1,2 +1,280 @@
 # \[Draft] How to contribute to AppFlowy
```

Comment:
Can you run through the title sections of this article with a title case tool? I believe that some of the titles do not have grammatically correct capitalization.

https://capitalizemytitle.com/

Reply:
Sure 👍